README.md

```diff
@@ -1,7 +1,67 @@
-## Notes
 
+# Zedd-App 
+>Custom time-tracking app with some very specific features. Built with Electron, TypeScript, React, MobX and MUI
+
+## General Information 📝
+- The application is designed to simplify work in the Clarity system
+- It operates as an interactive calendar, where you can schedule tasks and manage your work time efficiently
+- It has a user-friendly interface and features
```

Comment:
Diesen Punkt entfernen, denn jedes Produkt sollte, user-friedly sein
- It has a user-friendly interface and features